sixtyfps_runtime/rendering_backends/qt/qt_window.rs

```diff
@@ -176,6 +176,13 @@ cpp! {{
         }
     };
 
+    class SixtyFPSPublicGraphicsBlurEffect : public QGraphicsBlurEffect {
```

Comment:
(I would just have put this class in the function that uses it that in the function which uses it.)